components/Games/Checkers/Square/Square.tsx

```diff
@@ -1,14 +1,23 @@
 import cn from 'classnames';
 
+import { isJumpMove } from 'helpers/utils';
 import {SquarePropsI} from './SquareProps';
 import styles from './Square.module.scss';
 import React from "react";
 
 
 export const Square: React.FC<SquarePropsI> = (
-    {value, selected, onClick, disputable, number, flip}
+    {value, selected, onClick, disputable, number, flip, lastMove, onHandleMove}
 ) => {
     // console.log('Square disputable: ', disputable);
```

Comment:
It'd be nice to have a bit more info in this kind of log messages, even if they are commented out for potential future bug hunt.